lib/dockerCompose.ts

```diff
@@ -0,0 +1,106 @@
+import { ActiveDeployment } from './deployment';
+import { RA } from './typescriptCommonTypes';
+
+export const createDockerConfig = (
+  deployments: RA<ActiveDeployment>,
+  nginxConfigHash: number
+): string => `
+version: '3.9'
+services:
+
+${
+  deployments.length === 0
+    ? `
+  mariadb:
+    restart: unless-stopped`
+    : ''
+}
+${deployments
+  .map(
+    (deployment) => `
+  ${deployment.hostname}:
+    image: specifyconsortium/specify7-service:${deployment.branch}
+    init: true
```

Comment:
I think this might fix https://github.com/specify/specify7-test-panel/pull/4#issuecomment-960289216

```suggestion
    init: true
    networks:
       - nginx
```

Reply:
Thanks. Adding a few networks indeed fixed it. 38913ad
It event works on localhost, despite hostnames like `a11y.localhost` and `next-release.localhost` looking kind of unusual.